code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -1273,11 +1273,11 @@ nobiliumsuppression = INFINITY
 	if(cached_gases[/datum/gas/bz][MOLES] - consumed_amount < 0)
 		return NO_REACTION
 	if(cached_gases[/datum/gas/bz][MOLES] < 30)
-		radiation_pulse(location, consumed_amount, 1, TRUE, FALSE)
+		radiation_pulse(location, consumed_amount * 20, 1, TRUE, FALSE)
```

Comment:
To be totally clear, the third argument represents the range modifier of the wave. It's multiplied with the step count of the wave to figure out how fast the dropoff of radiation is per tile. it defaults to 1
`/// Higher than 1 makes it drop off faster, 0.5 makes it drop off half etc `
`strength = INVERSE_SQUARE(intensity, max(range_modifier*steps, 1), 1)`
If you want something that drops off faster, make it a 2 or a three.

Reply:
ohhh i thought it was something like max range! i'll change it to 2.5